Build/buen_aire_backend.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM amazonlinux:2
+
+# CLI utilities
+RUN yum install -y gcc git make awscli
+
+RUN yum install -y python3-devel
+
+ADD requirements.txt /requirements/
+
+# Python 3 code dependencies
+RUN pip3 install -r /requirements/requirements.txt
+
+WORKDIR /Buen-Aire-Backend
```

Comment:
Add a new line at the end of the file homie

Reply:
OK.

Reply:
I think I just said that to demo something to you guys when we talked about git, lol.